FEXCore/Source/Interface/Core/JIT/JITClass.h

```diff
@@ -341,6 +341,8 @@ class Arm64JITCore final : public CPUBackend, public Arm64Emitter {
 
   void EmitInterruptChecks(bool CheckTF);
 
+  void EmitEntryPoint(ARMEmitter::BackwardLabel* HeaderLabel, bool CheckTF);
```

Comment:
`HeaderLabel` can never be nullptr, so it should be a reference parameter for clarity.